examples/gitlab-centos7-atomicapp/Nulecule

```diff
@@ -0,0 +1,63 @@
+---
+specversion: 0.0.2
+id: gitlab-atomicapp
+
+metadata:
+  name: Gitlab App
+  appversion: 1.2.0
+  description: Gitlab.
+graph:
+  - name: redis
+    params:
+      - name: image
+        description: Redis image to be used.
+        default: swordphilic/redis:latest
+    artifacts:
+      kubernetes:
+        - file://artifacts/kubernetes/redis-rc.json
+        - file://artifacts/kubernetes/redis-service.json
+      docker:
+        - file://artifacts/docker/redis-container-run
+  - name: postgresql
+    params:
+      - name: image
+        description: PostgreSQL image to be used.
+        default: swordphilic/postgresql:latest
+      - name: DB_USER
+        description: Database user (This parameter is used for Docker provider)
```

Comment:
Why is the comment about Docker provider here? I see all the params used in Kubernetes manifests also..